fresque/__init__.py

```diff
@@ -10,15 +10,15 @@
 import logging.handlers
 import os
 import sys
-
 import flask
 import flask_fas_openid
-
+from inspect import getmembers, isfunction
 
 APP = flask.Flask(__name__)
+
 APP.config.from_object('fresque.default_config')
 if 'FRESQUE_CONFIG' in os.environ: # pragma: no cover
-    APP.config.from_envvar('FRESQUE_CONFIG')
+     APP.config.from_envvar('FRESQUE_CONFIG')
```

Comment:
@pypingou can you suggest me a linter to avoid these kind of error? The app is still running with above indentation.